.gitlab-ci.yml

```diff
@@ -1,22 +0,0 @@
-stages:
```

Comment:
@M-ZubairAhmed This is still used 🚀 but it's going to be removed hopefully in the next month

Reply:
ah i see sorry i reverted the file :)